packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -301,6 +320,12 @@ class PaginatedDataTableState extends State<PaginatedDataTable> {
     }
   }
 
+  @override
+  void reassemble() {
```

Comment:
Overriding this method is pretty rare (I think Image is the only other widget that does it). A comment explaining the what and why would be helpful.

Reply:
done

Reply:
NICE, more interesting than I expected!